crest-nextapp/app/(default)/(navPages)/Training/page.tsx

```diff
@@ -1,35 +1,49 @@
 import CommonPageComponent from "@/app/component/commonPageComponent";
 import React from "react";
+import { getTrainingCard } from "@/sanity/sanity-utils";
 
-const dummyData = [
-  {
-    title: "Open source license compliance",
-    pageName: "LCD-Seminar",
-    description:
-      "Learn how open source affects your business and develop an open source strategy for your organization! For decision makers in all industries and those consulting to them..",
-    imageUrl: "https://i0.wp.com/bayave.com/wp-content/uploads/2023/05/Image-LCD-Seminar.png?w=1728&ssl=1",
-  },
-  {
-    title: "Open source license compliance",
-    pageName: "OSB-Seminar",
-    description:
-      "Learn how to streamline license compliance of projects and products that contain open-source code! Critical skills and knowledge for everyone involved with open-source software..",
-    imageUrl: "	https://i0.wp.com/bayave.com/wp-content/uploads/2023/05/Image-OSB-Seminar.png?w=1728&ssl=1",
-  },
-  // Add more data as needed
-];
+// const dummyData = [
```

Comment:
I think the good practice is to remove all these comments to make the code cleaner. If we need these code later, we can easily track it from the commit history.

Reply:
great advice for good practice, will modify it after the meeting

Reply:
You can also remove it before the merge, so you do not need to raise a new PR later. By the way, good work boss!